packages/cojson/src/storage/DeletedCoValuesEraserScheduler.ts

```diff
@@ -0,0 +1,101 @@
+export type DeletedCoValuesEraserSchedulerRunResult = {
+  hasMore: boolean;
+};
+
+export type DeletedCoValuesEraserSchedulerOpts = {
+  throttleMs: number;
+  startupDelayMs: number;
+  followUpDelayMs: number;
+};
+
+type SchedulerState =
+  | "idle"
+  | "startup_scheduled"
+  | "throttle_scheduled"
+  | "followup_scheduled"
+  | "running"
+  | "disposed";
+
+export class DeletedCoValuesEraserScheduler {
+  private readonly runOnce: () => Promise<DeletedCoValuesEraserSchedulerRunResult>;
```

Comment:
`runOnce` name makes me think it will run only one time. Problably it is better something like `runJob` WDYT?